packages/language-server/src/xml-view-diagnostics.ts

```diff
@@ -142,7 +159,7 @@ function validationIssuesToLspDiagnostics<
         };
       /* istanbul ignore next - defensive programming */
       default:
-        assertNever(issueKind);
+        assertNever(issueKind as never);
```

Comment:
The point of this line was to check if all the cases are handled and we shouldn't use type cast here.